service/internal/env/env.go

```diff
@@ -2,12 +2,19 @@ package env
 
 import (
 	"errors"
+	"os"
+	"strings"
 	"sync"
 
 	"github.com/kelseyhightower/envconfig"
 )
 
-const appPrefix = "ARGO_CLOUDOPS"
+func init() {
```

Comment:
This might be my lack of go knowledge showing but does this get automatically called somewhere/somehow?

Reply:
Yup, on package load.